### PR TITLE
NL-110

### PR DESCRIPTION
**Pull request by geshan**
- [x] Added more tag to other posts too

### CODE REVIEW

File: source/_posts/2015-10-08-newsletter-number-110.markdown [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+title: "Newsletter #110"
+date: 2015-10-08 12:57
+author: Geshan Manandhar
+comments: true
+categories: [ db, mysql, angularjs, code, css, docker, dumps, easyvpn, es6, http2, js, json, naming, php, reactjs, refactoring, tdd, testing]
+---
+
+> Hi There!
+>
+> This week we got a new table tennis a.k.a ping pong table woo hoo!
```
Comment (razanbilwani) [inline source/_posts/2015-10-08-newsletter-number-110.markdown]: table tennis table.. too much alliteration, I would just go for 'we got a new ping pong table'

Comment (geshan) [inline source/_posts/2015-10-08-newsletter-number-110.markdown]: ok grammar guru.

**Comment by odino** [comment]
:+1:

**Comment by geshan** [comment]
going live with it.

**Pull request merged**